### PR TITLE
update markdown styling docs

### LINKED ISSUE #3485: Markdown content styling in a Markdown Layout does not match spec in documentation
### What version of `astro` are you using?

2.6.2

### Are you using an SSR adapter? If so, which one?

None

### What package manager are you using?

npm

### What operating system are you using?

Linux

### What browser are you using?

Firefox

### Describe the Bug

When using [a Markdown frontmatter layout](https://docs.astro.build/en/guides/markdown-content/#frontmatter-layout), the [documentation describes](https://docs.astro.build/en/guides/styling/#markdown-styling) being able to use `<style>` tags OR imported stylesheets to style markdown content.  However, the only way the markdown content is _actually_ styled is through the use of imported stylesheets.

This could be mitigated by explaining the difference more clearly in the documentation or enabling styling markdown content in a layout using `<style>` tags as described in the documentation.

### Link to Minimal Reproducible Example

https://stackblitz.com/edit/github-aks7dp?file=src%2Flayouts%2FWorkLayout.astro

### Participation

- [X] I am willing to submit a pull request for this issue.

### PR DESCRIPTION
**Pull request by kevinthenet**
#### What kind of changes does this PR include?

- New or updated content

#### Description

- Closes #3485  
- Rephrases the Markdown Styling documentation to better match the behavior of markdown content as it relates to markdown layout components.  See above issue for specifics.

### CODE REVIEW

File: src/content/docs/en/guides/styling.mdx [hunk 1]
```diff
@@ -520,11 +520,11 @@ Svelte in Astro also works exactly as expected: [Svelte Styling Docs][svelte-sty
 
 ## Markdown Styling
 
-Any Astro styling methods are available to a [Markdown layout component](/en/core-concepts/layouts/#markdownmdx-layouts).
+Any Astro styling methods are available to a [Markdown layout component](/en/core-concepts/layouts/#markdownmdx-layouts). Please note that layout styling is not the same as the rendered Markdown content styling.
```
Comment (sarah11918) [inline src/content/docs/en/guides/styling.mdx:523]:
```suggestion
Any Astro styling methods are available to a [Markdown layout component](/en/core-concepts/layouts/#markdownmdx-layouts), but different methods will have different styling effects on your page.
```
original sentence unclear

File: src/content/docs/en/guides/styling.mdx [hunk 2]
```diff
@@ -520,11 +520,11 @@ Svelte in Astro also works exactly as expected: [Svelte Styling Docs][svelte-sty
...
 
-You can apply global styles to your Markdown content by adding [`<style>` tags](#global-styles) and [imported stylesheets](#external-styles) to the layout that wraps your page content. You can also add [CSS integrations](#css-integrations) including [Tailwind](/en/guides/integrations-guide/tailwind/).
+You can apply global styles to your Markdown content by adding [`<style>` tags](#global-styles) and/or [imported stylesheets](#external-styles) to the layout that wraps your page content. Imported stylesheets are the simplest way to only apply stylings to rendered page content, however are still subject to [Astro's cascading order](#cascading-order).
```
Comment (sarah11918) [inline src/content/docs/en/guides/styling.mdx:525]:
```suggestion
You can apply global styles to your Markdown content by adding [imported stylesheets](#external-styles) to the layout that wraps your page content. It is also possible to style your Markdown with [`<style is:global>` tags](#global-styles) in the layout component.  Note that any styles added are subject to [Astro's cascading order](#cascading-order), and you should check your rendered page carefully to ensure your styles are being applied as intended.
```
We don't describe methods as "simple" or "easy" in docs. But if it represents a happier path, we can put it first, as its own statement, and make the global style tags a bit of an afterthought. I've also added the global attribute here match the wording of "global styles."

**Review by sarah11918 — COMMENTED** [review]
Thanks for picking up the issue @kevinthenet !

I've made a couple of tiny changes below to better fit with our docs, and happy to welcome you to Team Docs!

File: src/content/docs/en/guides/styling.mdx (hunk 1 above)
Comment (kevinthenet) [reply src/content/docs/en/guides/styling.mdx]: I'm fine with the change as is, but the original change proposed was meant to emphasize the difference between the Markdown Layout component styling and the styling for rendered markdown content.

Let me know what you think about this:

```suggestion
Any Astro styling methods are available to a [Markdown layout component](/en/core-concepts/layouts/#markdownmdx-layouts), but may not apply to styling for rendered markdown content.
```

**Pull request merged**